pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch_test.go

```diff
@@ -724,34 +736,32 @@ func TestApplyListenerPatches(t *testing.T) {
 // This benchmark measures the performance of Telemetry V2 EnvoyFilter patches. The intent here is to
 // measure overhead of using EnvoyFilters rather than native code.
 func BenchmarkTelemetryV2Filters(b *testing.B) {
```

Comment:
Why is this ~3x faster now? The proxy IS a match, so shouldn't there be no improvement?

Reply:
Actually the most important part is i moved the `proxyMatch` to the `PushContext.EnvoyFilters`, which is actually called once per proxy. But currently, the times is correlated to the number of listeners, patches, filterchains, networkfilters, etc.